packages/web-components/fast-components/src/anchored-region/anchored-region.stories.ts

```diff
@@ -46,60 +45,59 @@ addons.getChannel().addListener(STORY_RENDERED, (name: string) => {
-                ) {
-                    togglesRegion!.setAttribute("horizontal-default-position", "left");
-                } else {
-                    togglesRegion!.setAttribute("horizontal-default-position", "right");
-                }
-            });
-
-        document
-            .getElementById("toggle-positions-vertical")!
-            .addEventListener("click", (e: MouseEvent) => {
-                if (togglesRegion?.getAttribute("vertical-default-position") === "top") {
-                    togglesRegion!.setAttribute("vertical-default-position", "bottom");
-                } else {
-                    togglesRegion!.setAttribute("vertical-default-position", "top");
-                }
-            });
-
-        document
-            .getElementById("toggle-inset-vertical")!
-            .addEventListener("click", (e: MouseEvent) => {
-                if (togglesRegion?.getAttribute("vertical-inset") === "true") {
-                    togglesRegion!.setAttribute("vertical-inset", "false");
-                } else {
-                    togglesRegion!.setAttribute("vertical-inset", "true");
-                }
-            });
-
-        document
-            .getElementById("toggle-inset-horizontal")!
-            .addEventListener("click", (e: MouseEvent) => {
-                if (togglesRegion?.getAttribute("horizontal-inset") === "true") {
-                    togglesRegion!.setAttribute("horizontal-inset", "false");
-                } else {
-                    togglesRegion!.setAttribute("horizontal-inset", "true");
-                }
-            });
+        [
+            {
+                id: "toggle-positions-horizontal",
+                attribute: "horizontal-default-position",
+                firstValue: "right",
+                secondValue: "left",
+            },
+            {
+                id: "toggle-positions-vertical",
+                attribute: "vertical-default-position",
+                firstValue: "top",
+                secondValue: "bottom",
+            },
+            {
+                id: "toggle-inset-vertical",
+                attribute: "vertical-inset",
+                firstValue: "true",
+                secondValue: "false",
+            },
+            {
+                id: "toggle-inset-horizontal",
+                attribute: "horizontal-inset",
+                firstValue: "true",
+                secondValue: "false",
+            },
+        ].forEach(({ id, attribute, firstValue, secondValue }) => {
+            const toggleElement = document.getElementById(id);
+            if (toggleElement) {
+                toggleElement.addEventListener("click", (e: MouseEvent) => {
+                    if (togglesRegion?.getAttribute(attribute) === firstValue) {
+                        togglesRegion.setAttribute(attribute, secondValue);
+                    } else {
+                        togglesRegion.setAttribute(attribute, firstValue);
+                    }
```

Comment:
You don't need a check for togglesRegion, it's checked above. You can can setAttribute once here.
```suggestion
                    togglesRegion.setAttribute(attribute, togglesRegion.getAttribute(attribute) === firstValue ? secondValue : firstValue);
```